src/compiler/diagnosticMessages.json

```diff
@@ -2352,6 +2352,10 @@
         "category": "Error",
         "code": 4092
     },
+    "Extends clause of exported class '{0}' refers to a type whose name cannot be referenced.": {

```

Comment:
Are you sure? All the other messages like this just start with "Extends clause ...".